happy.plantuml

```diff
@@ -0,0 +1,42 @@
+@startuml
```

Comment:
Github supports mermaid diagrams natively, if that's easier than rendering images from plantUML.

Reply:
Oh very nice, good to know. I'll look into this next time :)